### ISSUE TITLE
Recursive http_archive

### ISSUE THREAD

**Issue by lshamis** [bazelbuild/bazel#3941]
I'm using `http_archive` to add https://github.com/abseil/abseil-cpp to my WORKSPACE.
When trying to use one of their BUILD rules, I get the following error:

    .../external/com_google_absl/absl/time/BUILD.bazel:27:1: no such package '@com_googlesource_code_cctz//': The repository could not be resolved and referenced by '@com_google_absl//absl/time:time'

`com_googlesource_code_cctz` is defined abseil's WORKSPACE as an `http_archive`.

**Comment by buchgr** [comment]
cc @damienmg

**Comment by damienmg** [comment]
Blocked by #1943

**Comment by dslomov** [comment]
This is actually a duplicate of #1943, closing
